xla/backends/gpu/runtime/command_buffer_thunk_test.cc

```diff
@@ -1425,6 +1433,143 @@ TEST(CommandBufferThunkTest, WhileCmd) {
   ASSERT_EQ(dst, std::vector<int32_t>(4, 15));
 }
 
+TEST(CommandBufferThunkTest, CuDnnCmd) {
```

Comment:
Hey @Cjkkkk,
we are trying to move to a single build graph where you can build CUDA and ROCm things with the same Blaze config. To help with that it would be great if you could move CUDA specific tests into a separate file - something like `cuda_command_buffer_thunk_test.cc` and then tag the xla_test target as `cuda-only`.

Reply:
should be fixed now. could you take another look?